### PR TITLE
Create footer and add color

### PR DESCRIPTION
**Pull request by alphaX86**
Added footer component. Requesting for review......

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import React from "react"
 const Header = ({ siteTitle }) => (
   <header
     style={{
-      background: `rebeccapurple`,
+      background: `#440047`,
```
Comment (alphaX86) [inline src/components/header.js:8]: Added color violet to header

File: src/components/footer.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Footer.propTypes = {
+    siteTitle: PropTypes.string,
+}
+
+Footer.defaultProps = {
+    siteTitle: ``,
+  }
+  
+export default Footer
+
```
Comment (alphaX86) [inline src/components/footer.js:43]: Created footer component

**Comment by alphaX86** [comment]
Just see and change it bro! Good night 🌃

File: package.json [hunk 3]
```diff
@@ -14,9 +14,9 @@
     "gatsby-source-filesystem": "^2.3.25",
     "gatsby-transformer-sharp": "^2.5.13",
     "prop-types": "^15.7.2",
-    "react": "^16.12.0",
+    "react": "^16.13.1",
     "react-bootstrap": "^1.3.0",
-    "react-dom": "^16.12.0",
+    "react-dom": "^16.13.1",
```
Comment (alphaX86) [inline package.json:19]: Changed versions to latest

**Review by BrightonDube — APPROVED** [review]
Great

**Pull request merged**